packages/block-library/src/spacer/editor.scss

```diff
@@ -3,5 +3,6 @@
 }
 
 .block-library-spacer__resize-container {
+	clear: both;
```

Comment:
Should we add an equivalent style on the frontend?

Reply:
Added in 7d5523b9a349bfbe57408753afe02f3398b943c8. ✔️